app/assets/stylesheets/errors.css

```diff
@@ -0,0 +1,84 @@
+.field_with_errors input,
+.field_with_errors select,
+.field_with_errors textarea {
+  background-color: pink;
+  border: 0.25em solid red;
+  font-weight: bold;
+}
+
+#errorExplanation,
+#error_explanation {
+  border: 3px solid red;
+  padding: 0.85em;
+  padding-bottom: 0;
+  margin-bottom: 1em;
+  background-color: pink;
+}
+
+#errorExplanation h2,
+#error_explanation h2 {
+  text-align: left;
+  font-weight: bold;
+  padding: 0.25em 0.25em 0.25em 1.5em;
+  font-size: 0.9em;
+  margin: -7px;
+  margin-bottom: 0;
+}
+
+#errorExplanation p {
```

Comment:
Same as below, everything #errorExplanation might already be handled by Foundation.

Reply:
This is the section I needed for Devise.

Reply:
see #571